### PR TITLE
Added forgotten parameter

### PR DESCRIPTION
**Pull request by miloit**
Added forgotten parameter

### CODE REVIEW

File: src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h [hunk 1]
```diff
@@ -85,6 +85,7 @@ class ListTvChannelModel : public QAbstractListModel {
 class BrowseTvChannelModel : public QObject {
     Q_OBJECT
     Q_PROPERTY(QString id READ id NOTIFY idChanged)
+    Q_PROPERTY(QString time READ time NOTIFY idChanged)
```
Comment (zehnm) [inline src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h]: Invalid signal, this should be timeChanged

**Review by zehnm — CHANGES_REQUESTED** [review]

File: src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h (hunk 1 above)
Comment (miloit) [reply src/yio-model/mediaplayer/tvchannelmodel_mediaplayer.h]: Fixed

**Review by zehnm — APPROVED** [review]
Looks good to me.
Please check @martonborzak if there's anything else required when adding a new command enum. For the future: wouldn't it be better to add new enums in the end? Otherwise this will be a breaking change.
Note to myself & @martonborzak: with the other recent interface changes it's no longer backwards compatible and we have to release a 1.0 version as soon as I have the dependency checks in the plugin loader. Otherwise the app will just crash if there's another plugin compiled with an older intg library!

**Review by martonborzak — APPROVED** [review]
I think that's all for commands. For features you need to add it to `config-schema.json` as well.

**Pull request merged**